config/src/config_wrapper.cpp

```diff
@@ -72,21 +56,146 @@ void load()
     dynamic_config.load_from_file("dynamic_config.json");
 
     loaded = true;
+    log << "Loaded files";
+}
+
+int32_t number_of_stations()
+{
+    load();
+    try {
+        static auto tmp = static_config.getSize("stations") + static_config.getSize("end_stations");
+        return tmp;
+    }
+    catch (const std::exception &e) {
+        log << "number_of_stations";
```

Comment:
Måske prefixe med at det er en fejl?
```suggestion
        log << "error: number_of_stations";
```

Reply:
denne kommentar er generel til fejlbeskeder. Evt. kan man lave et proxy object i log.hpp hvis eneste formål er at indikere typen af besked, så det bliver lidt mindre hardcoded.

Reply:
:question: til sidste del.